receiver/awscontainerinsightreceiver/internal/neuron/neuron_monitor_scraper_config.go

```diff
@@ -0,0 +1,100 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package nueron
+
+import (
+	"time"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/prometheusscraper"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/config"
+	"github.com/prometheus/prometheus/discovery"
+	"github.com/prometheus/prometheus/discovery/kubernetes"
+	"github.com/prometheus/prometheus/model/relabel"
+)
+
+const (
+	caFile             = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	collectionInterval = 60 * time.Second
+	jobName            = "containerInsightsNeuronMonitorScraper"
+)
+
+func GetNueronScrapeConfig(opts prometheusscraper.SimplePromethuesScraperOpts) *config.ScrapeConfig {
+
+	return &config.ScrapeConfig{
+		ScrapeInterval: model.Duration(collectionInterval),
+		ScrapeTimeout:  model.Duration(collectionInterval),
+		JobName:        jobName,
+		Scheme:         "http",
+		MetricsPath:    "/metrics",
+		ServiceDiscoveryConfigs: discovery.Configs{
+			&kubernetes.SDConfig{
+				Role: kubernetes.RoleService,
+				NamespaceDiscovery: kubernetes.NamespaceDiscovery{
+					IncludeOwnNamespace: true,
+				},
+				Selectors: []kubernetes.SelectorConfig{
+					{
+						Role:  kubernetes.RoleService,
+						Label: "k8s-app=neuron-monitor-service",
+					},
+				},
+				AttachMetadata: kubernetes.AttachMetadataConfig{
+					Node: true,
+				},
```

Comment:
Sorry didn't notice this before, but do you need this?  What effect does it have?